### ISSUE TITLE
[NPC] Heroic Training Dummy stats

### ISSUE THREAD

**Issue by Crytexx** [dalaranwow/dalaran-wow#4669]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Training dummies can not parry or dodge.

**Expected behaviour**: Training dummies were introduced to simmulate personal caps against bosses.
They should dodge, parry, and miss. Heroic training dummy should have same stats as lvl 83 boss, meaning: 24% glancing blow chance, 10643 base armor,  6.50% dodge from behind,...,so on.

**Steps to reproduce the problem**:

1. remove any expertise you have equipped
2. hit dummy from front; hit dummy from back
3. No dodges observed no matter the time

**Include proofs for this behaviour**

http://www.wowhead.com/npc=31146/raiders-training-dummy#comments

**Comment by TeridanDW** [comment]
Confirmed, they also have core script that can be looked at.

> By PrettyKitty (3,327 – 1·4·18) on 2008/12/24 (Patch 3.0.3)		
> The odd thing is that these things can dodge, parry, and miss.
> I mean they are just sitting non-living dummies how could this be 

On a side note they do have exactly 10673 armor ingame now.

**Comment by Winstrol** [comment]
Also seems like lots of procs doesnt trigger when attacking a target dummy

**Comment by Crytexx** [comment]
Exactly. Like, I do not want to be pain in the ass, when there is so many class bugs and so on, but...
This should be addressed as priority fix, because it would make class testing such more easier.

**Comment by Guzzone** [comment]
It would be cool to have this fixed especially now since Expertise has been fixed (at least accordingly to the bug tracker) so people would be able to test if the fix is actually working or not.
